docs/src/tool/index.md

```diff
@@ -105,18 +105,24 @@ named `default`.
 + The environment variable `REACH_DEBUG`, if set to any non-empty value, enables debug messages from the Reach compiler, which will appear in the console.
   This debug information includes: the estimated cost of the contract on Algorand.
   This variable automatically enabled `--intermediate-files`.
+  
+  A `reach compile` usage example is available in [Overview](##over-compile). 
```

Comment:
I believe this indentation makes it go with the list item, but you don't want that